src/checks/one-relearn-move.js

```diff
@@ -0,0 +1,15 @@
+const gameCheck = require('../helpers/gameId.js');
+
+module.exports = {
+  description: 'Pokemon hatched from an egg in gen 6-7 must have at least one relearn move.',
+  field: 'eggMoveName',
```

Comment:
For fields that are multiple named ones, we really should have an umbrella one that it points to. IT's wrong to set this to 1 of them that it might not be the one that is wrong.